docs/man/rpm-common.8.scd

```diff
@@ -0,0 +1,229 @@
+	for the common macro definition needs.
+
+*--quiet*
+	Print as little as possible - normally only error messages will be
+	displayed.
+
+*--rcfile* _FILELIST_
+	Replace the default list of configuration files to be read with
+	_FILELIST_.  See *rpmrc Configuration* for details.
+
+*--root* _DIRECTORY_
+	Use the file system tree rooted at _DIRECTORY_ for all operations.
+	Note that this means the database within _DIRECTORY_ will be used
+	for dependency checks and any scriptlet(s) (e.g. *%post* if
+	installing, or *%prep* if building, a package) will be run after a
+	chroot(2) to _DIRECTORY_.
+
+	Note that rpm assumes the environment inside the root is set up by
+	the caller, such as any mounts needed for the operation inside the
+	root directory.
+
+*--undefine*='_MACRO_'
+	Undefines _MACRO_.
+
+*-v*,
+*--verbose*
+	Print verbose information - normally routine progress messages will
+	be displayed. Supply more than once to increase verbosity, eg. *-vv*.
+
+# DEBUG OPTIONS
+*--debug*
+	Print lots of debug information.
+
+*--fsmdebug*
+	Print debuging information of payload handling code.
+
+*--rpmiodebug*
+	Print debug information about file IO.
+
+*--stats*
+	Print runtime statistics of often used functions.
+
+# DEBUG OPTIONS
+Options which are only useful for troubleshooting.
+
+*--deploops*
+	Print dependency loops as warning.
+
+*--fsmdebug*
+	Print debuging information of payload handling code.
+
+*--rpmiodebug*
+	Print debug information about file IO.
+
+*--stats*
+	Print runtime statistics of often used functions.
+
+# FTP/HTTP OPTIONS
+These options are deprecated. Use special purpose software such as
+*curl*(1) or *wget*(1) for fetching data from the internet.
+
+For some operations, *rpm* can act as an FTP and/or HTTP client so that
+packages can be queried or installed from the internet. Package files for
+install, upgrade, and query operations may be specified as an *ftp* or *http*
+style URL:
+
+_http://HOST[:PORT]/path/to/package.rpm_
+
+_ftp://[USER:PASSWORD]@HOST[:PORT]/path/to/package.rpm_
+
+If both the user and password are omitted, anonymous *ftp* is used.
+
+*rpm* allows the following options to be used with ftp URLs:
+
+*rpm* allows the following options to be used with *http* and *ftp* URLs:
+
+*--httpproxy* _HOST_
+	The host _HOST_ will be used as a proxy server for all *http* and
+	*ftp* transfers. This option may also be specified by configuring
+	the macro *%\_httpproxy*.
+
+*--httpport* _PORT_
+	The TCP _PORT_ number to use for the *http* connection on the
+	proxy http server instead of the default port. This option may also
+	be specified by configuring the macro *%\_httpport*.
+
+# ENVIRONMENT
+*RPM_CONFIGDIR*
+	Used to override the default *rpm* configuration home,
+	typically _/usr/lib/rpm_.
+
+# EXIT STATUS
+On success, 0 is returned, a non-zero failure code otherwise.
+
+# FILES
+
+## rpmrc Configuration
+
+Each file in the colon separated rpmrc path is read sequentially by
```

Comment:
"rpmrc" is not obivous in this context. Is that the `--rcfile` option?

Reply:
Hmm, this might be tricky to reword since there's the "default rpmrc path" later in this section as well...

Reply:
Yup, I basically just lifted the existing text to a different place without even trying to improve it. 
rpmrc does of course relate to `--rcfile` too, but more importantly the default rpmrc's and the path they're looked through forms the other base pillar of rpm configuration in addition to macros.

I'd leave better explaining the system to #3614, this is more about reorganizing to match the new style rather than fixing the contents too, although some content-fixing is necessary as well.